package net.my.user;

public class UserFoundException extends Exception {

}
